tests/test_quantization_metric.py

```diff
@@ -0,0 +1,210 @@
+                 'signed': 75.0, 'asymmetric': 75.0, 'in all': 75.0},
+            '% AQ of number spaced AQ': {'per channel': 0.0, 'symmetric': 0.0, 2: 16.666666666666664, 4: 0.0,\
+                 8: 83.33333333333334, 'signed': 100.0, 'asymmetric': 100.0, 'in all': 100.0},
+            '% Q of number total potential Q': {'per channel': 31.25, 'symmetric': 31.25, 2: 12.5, 4: 6.25, 8: 50.0,\
+                 'signed': 37.5, 'asymmetric': 37.5, 'in all': 68.75},
+            '% WQ of number potential WQ': {'per channel': 62.5, 'symmetric': 62.5, 2: 12.5, 4: 12.5, 8: 37.5,\
+                 'signed': 0.0, 'asymmetric': 0.0, 'in all': 62.5},
+            '% WQ of number spaced WQ': {'per channel': 100.0, 'symmetric': 100.0, 2: 20.0, 4: 20.0, 8: 60.0,\
+                 'signed': 0.0, 'asymmetric': 0.0, 'in all': 100.0},
+            'params': {'per channel': 0, 'symmetric': 0, 2: 0, 4: 0, 8: 0, 'signed': 0, 'asymmetric': 0,\
+                 'in all': 0}}),
+    TestStruct(
+        initializers={"precision": {
+            "bitwidth_per_scope":
+            [[2, 'AlexNet/Sequential[features]/NNCFConv2d[0]'],
+             [4, 'AlexNet/Sequential[features]/NNCFConv2d[6]']]
+        }},
+        activations={"bits": 8},
+        weights={"bits":6},
+        ignored_scopes=['AlexNet/Sequential[classifier]'],
+        table={
+            '% AQ of number potential AQ': {'per channel': 0.0, 2: 12.5, 4: 0.0, 6: 0.0, 8: 62.5, 'symmetric': 75.0,\
+                 'in all': 75.0, 'signed': 0.0, 'asymmetric': 0.0},
+            '% AQ of number spaced AQ': {'per channel': 0.0, 2: 16.666666666666664, 4: 0.0, 6: 0.0,\
+                 8: 83.33333333333334, 'symmetric': 100.0, 'in all': 100.0, 'signed': 0.0, 'asymmetric': 0.0},
+            '% Q of number total potential Q': {'per channel': 0.0, 2: 12.5, 4: 6.25, 6: 18.75, 8: 31.25,\
+                 'symmetric': 68.75, 'in all': 68.75, 'signed': 0.0, 'asymmetric': 0.0},
+            '% WQ of number potential WQ': {'per channel': 0.0, 2: 12.5, 4: 12.5, 6: 37.5, 8: 0.0, 'symmetric': 62.5,\
+                 'in all': 62.5, 'signed': 0.0, 'asymmetric': 0.0},
+            '% WQ of number spaced WQ': {'per channel': 0.0, 2: 20.0, 4: 20.0, 6: 60.0, 8: 0.0, 'symmetric': 100.0,\
+                 'in all': 100.0, 'signed': 0.0, 'asymmetric': 0.0},
+            'params': {'per channel': 0, 2: 0, 4: 0, 6: 0, 8: 0, 'symmetric': 0, 'in all': 0, 'signed': 0,\
+                 'asymmetric': 0}}),
+    TestStruct(
+        initializers={"precision": {
+            "bitwidth_per_scope":
+            [[2, 'AlexNet/Sequential[features]/NNCFConv2d[0]'],
+             [4, 'AlexNet/Sequential[features]/NNCFConv2d[6]']]
+        }},
+        activations={"bits": 8},
+        weights={"bits":6},
+        ignored_scopes=[],
+        table={
+            '% AQ of number potential AQ': {'per channel': 0.0, 2: 12.5, 4: 0.0, 6: 0.0, 8: 87.5, 'symmetric': 100.0,\
```

Comment:
So if we decide to change a display name of a column in the table, we will have to do the rename in all tests as well? Doesn't seem very robust or convenient to me.